### DISCUSSION TITLE
react-parallax package video usage

### DISCUSSION THREAD

**Discussion by WayneMacMavis** [npm/feedback#265] [General]
I'm not sure if this is the place to ask this question. But I have exhausted all other options and I still do not have the answer to my question.

I've been trying to make a parallax effect with my video background using the react-parallax package. The bgImage tag only accepts a image and not a video. So I thought that there would be a way to achieve this using the background tags. But still no luck, Am I missing something? Or is it not possible to use react-parallax with a background video?

Any help would be greatly appreciated.

**Comment by MylesBorins** [comment]
@WayneMacMavis this repo is for making requests for new features in npm products such as the CLI / Registry / Website. Unfortunately we don't offer general support for packages found on the npm registry

You might have luck with http://github.community/ or https://stackoverflow.com/ as a place to ask support questions, apologies that I can't help you more
